lumen/ai/agents.py

```diff
@@ -391,9 +393,16 @@ def format_float(num):
             else:
                 df_head_dict[col] = df_head_dict[col].tolist()
 
-        data = {"stats": df_describe_dict, "head": df_head_dict}
-        data_string = str(data)
-        return data_string
+        data = {
+            "summary": {
+                "total_length": length,
+                "is_summarized": is_summarized,
+                "dtypes": {col: str(dtype) for col, dtype in df.dtypes.to_dict().items()}
```

Comment:
For dtypes, I did:
```python
        # df_dtypes_dict = {
        #     col: str(type(df[col].iloc[:1].astype("object").iloc[0])).split("'")[1]
        #     for col in df.columns
        # }
```
To reduce the number of tokens and also make it into native python dtypes if I recall correctly (e.g. dtype('int64') -> int)

Reply:
Please fix.